manifest.json

```diff
@@ -13,6 +13,14 @@
             "matches": [
                 "*://*/*"
             ],
+			"include_globs": [
+				"*outlook*",
+				"*mail.google*",
+				"*groupme*",
+				"*linkedin*",
+				"*slack*",
+				"*yahoo*"
```

Comment:
Websites like https://www.yahoo.com/news/ and https://finance.yahoo.com/ are still included in this. 
Make sure you add support for "inbox.google.com" as well - it's another viewer for Gmail and lots of people in the JNS comments complained that inbox didn't work for them, but it can for us :)